HACKER/HACKER/Sources/ViewControllers/SettingViewController/SettingViewController.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  SettingViewController.swift
+//  HACKER
+//
+//  Created by Yi Joon Choi on 2022/03/24.
+//
+
+import UIKit
+import SnapKit
+import Then
+
+class SettingViewController: UIViewController {
+    
+    // MARK: - Components
+    private let navigationBar = HackerNavigationBar()
+    private lazy var settingsTableView = UITableView(frame: .zero, style: .grouped).then {
+        $0.dataSource = self
+        $0.delegate = self
+        $0.backgroundColor = .hackerWhite
+        $0.separatorStyle = .none
+        $0.register(SettingTableViewCell.self, forCellReuseIdentifier: SettingTableViewCell.identifier)
+        
+        if #available(iOS 15, *) {
+            $0.sectionHeaderTopPadding = 0
+        }
+    }
+    
+    private let settingTitles = ["닉네임 변경", "이용약관", "오픈소스 라이선스",
+                                 "만든 사람들", "로그아웃", "서비스 탈퇴"]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configUI()
+        setupAutoLayout()
+    }
+    
+    // MARK: - Custom Method
+    
+    private func configUI() {
+        self.view.backgroundColor = .hackerWhite
+        self.navigationController?.navigationBar.isHidden = true
+    }
+    
+    private func setupAutoLayout() {
+        view.addSubviews([navigationBar, settingsTableView])
+        navigationBar.iconLayout(logoImage: UIImage(named: "settingIcon"), rightImage: nil)
+        navigationBar.popViewController = {
+            self.navigationController?.popViewController(animated: true)
+        }
+        navigationBar.snp.makeConstraints { make in
+            make.top.leading.trailing.equalTo(self.view.safeAreaLayoutGuide)
+            make.height.equalTo(72)
+        }
+        settingsTableView.snp.makeConstraints { make in
+            make.top.equalTo(self.navigationBar.snp.bottom).offset(8)
+            make.leading.equalToSuperview().inset(24)
+            make.bottom.equalTo(view.safeAreaLayoutGuide)
+            make.centerX.equalToSuperview()
+        }
+    }
+}
+
+// MARK: - UITableViewDataSource
+extension SettingViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 1
+    }
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 6
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: SettingTableViewCell.identifier) as? SettingTableViewCell else { return UITableViewCell() }
+        
```

Comment:
설정뷰가 그냥 까만 네모들로 보이길래 왜 그런가 봤더니
컬렉션뷰 cell background 색 설정이 안 돼있어서 다크모드에서는 까맣게 보이고 있더라구
<img width="1192" alt="image" src="https://user-images.githubusercontent.com/71601985/160169542-37d4128f-c33c-48cb-aa91-e9703a0622cd.png">

설정해주자~!
```suggestion
        cell.backgroundColor = .clear
```

Reply:
세심 걸... 확인!!!